source/products/roland/haus-und-grund/webapp.rst

```diff
@@ -0,0 +1,21 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        touchpoint: "tp_14a00f394fe37d85348bce9908907",
+        mode: "fullscreen",
+        language: 'de',
+        metadata: {
+          tan: response.tan,
+        }
+      })
```

Comment:
As this webapp needs to always be loaded with prefill, please add there all needed prefill in snippet too.
Otherwise when PM or someone else will copy this snippet, webapp flow will not work.

Reply:
You can set just some placeholders/short description in values for those prefilled properties. For example:
```
living_module: {living module}
```